_templates/exchanges.html

```diff
@@ -63,6 +63,7 @@ <h3 id="brazil"><img src="/img/flags/BR.png" alt="Brazilian flag">Brazil</h3>
       <p>
         <a href="https://foxbit.com.br/">Foxbit</a><br>
         <a href="https://www.mercadobitcoin.com.br/">Mercado Bitcoin</a><br>
+        <a href="https://www.omnitrade.io/">OmniTrade</a>
```

Comment:
A line break needs to be added at the end of this line, otherwise OmniTrade and Walltime will appear on the same line.

Reply:
oh my Bad, forgot the `<br>`

now Is ok :)